src/msg/simple/DispatchQueue.cc

```diff
@@ -150,6 +150,14 @@ void DispatchQueue::entry()
       lock.Unlock();
 
       if (qitem.is_code()) {
+	if (g_conf->ms_inject_internal_delays > 0 &&
+	    g_conf->ms_inject_delay_probability &&
+	    (rand() % 1000) / 1000.0 > g_conf->ms_inject_delay_probability) {
```

Comment:
The inject probability judgement logic is a little bit confusing. The chance of injecting delay shall go high as the ms_inject_delay_probability increases, e.g., 
```
1. if ms_inject_delay_probability == 0, we don't inject delay;
2. if ms_inject_delay_probability == 1, we shall 100% inject delay;
```
It's quite the opposite logic here, especially when ms_inject_delay_probability is set to 1, we will absolutely **not** inject delay, as ```(rand() % 1000) / 1000.0```  is always smaller than 1.